### ISSUE TITLE
Preserve empty alt attribute value when minifying HTML

### ISSUE THREAD

**Issue by maxhoffmann** [Keats/gutenberg#1683]
# Bug Report

## Environment

Zola version: 0.14.1

## Expected Behavior
Minifying the html zola should preserve `alt=""` attributes, so that screen readers know that images are decorative elements.
 
## Current Behavior
Zola minifies `alt=""` attributes to just `alt`, so that screen readers (and Google Page Speed) think that alternate text is missing.

## Step to reproduce
Minify that includes an image tag with Zola, eg `<img src="some.png" alt="">`.

**Comment by Keats** [comment]
Can you open that issue upstream: https://github.com/wilsonzlin/minify-html?
Nothing that can be done from Zola itself.

**Comment by maxhoffmann** [comment]
@Keats Upstream version `0.7.1` should fix this issue 🙂

**Comment by Keats** [comment]
Cool, that will be updated for the next release of Zola then.
